debug.log

```diff
@@ -0,0 +1 @@
+[1106/192439.086:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Don't track these files
@IsuruLakshan97 add *.log in the .gitignore file

Reply:
done